static/js/containers/ProfilePage.js

```diff
@@ -40,9 +40,9 @@ class ProfilePage extends React.Component {
       dispatch(startProfileEdit());
     }
     return dispatch(validateProfile(profile, requiredFields, messages)).then(() => {
-      dispatch(saveProfile(SETTINGS.username, profile)).then(() => {
-        dispatch(clearProfileEdit());
-      });
+      return dispatch(saveProfile(SETTINGS.username, profile));
+    }).then(() => {
+      return dispatch(clearProfileEdit());
```

Comment:
Just a stylistic difference, I'll change it to what we're doing in #314 

Technically the `return dispatch(clearProfileEdit())` is different but we don't use the returned value anyway